packages/queries/src/getColumns.ts

```diff
@@ -0,0 +1,33 @@
+import { DbSchema } from './types/schemas';
+
+export function getColumns(schema: DbSchema, table: string) {
```

Comment:
Without any docs I would have assumed that a function called `getColumns(schema,table)` would return all the columns. That's not the case, though, it returns the selectable columns.

Naming your reflect the particularities of your function. In this case the important bit is that it's returning _only_ the columns that are selectable.

Reply:
Two suggestions:

1. Add some docs (ideally with an example)
2. Rename the function to `getSelectableColumns`